fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -740,7 +740,7 @@ public class Config extends ConfigBase {
      * txn manager will reject coming txns
      */
     @ConfField(mutable = true, masterOnly = true)
-    public static int max_running_txn_num_per_db = 100;
+    public static int default_db_max_running_txn_num = 100;
```

Comment:
How about NOT changing the origin config name?
As far as I know, many user modified this config by their own

Reply:
> How about NOT changing the origin config name? As far as I know, many user modified this config by their own

If we apply the new config, the former one is useless. 
And if we don't change the config name according to the existing format of database configs, there will be ambiguity.